styles/style.css

```diff
@@ -163,8 +206,24 @@ p {
 
 .article {
   margin-bottom: 1rem;
+  /*display: flex;
+  flex-direction: row;*/
+  border:#B8B8B8 1px solid;
+  padding: 1rem;
+}
+
+.aPair {
+  display: flex;
+
+
 }
 
+.bottom_articles {
+  display: flex;
+  /* only way I can see to place a suitable gap in between. However there must be another way? */
+  gap: 1rem;
```

Comment:
I actually didn't know this CSS property. I'd use margin on the article itself, but it will leave unwanted gaps on either left or right side. There is another solution using selectors like `first-of-type` and `last-of-type` as can be seen here https://stackoverflow.com/questions/8184958/add-space-between-html-elements-only-using-css but I like your simple solution better!